glean/glean.js

```diff
@@ -0,0 +1,2 @@
+//# sourceMappingURL=glean.js.map
```

Comment:
Hm, I wonder if Qt picks up on the source maps. Do you have any idea @bakulf ? If it doesn I can include the source map file in the release as well. That could be useful for debugging.

Reply:
QT does not support source maps (yet?).

Reply:
Might as well remove the source maps comment from this file in future releases of Glean.js either way :)